src/pages/iou/MoneyRequestSelectorPage.js

```diff
@@ -94,9 +92,8 @@ function MoneyRequestSelectorPage(props) {
                                 title={title[iouType.current]}
                                 onBackButtonPress={() => Navigation.goBack()}
                             />
-                            <TopTab.Navigator
-                                initialRouteName={props.selectedTab}
-                                backBehavior="order"
+                            <OnyxTabNavigator
```

Comment:
I see you took away `backBehavior="order"` ... was that intentional?

Reply:
Yes, it didn't seem to be working any different than the default